### ISSUE TITLE
Error on loading the colorscheme

### ISSUE THREAD

**Issue by Glavnokoman** [marko-cerovac/material.nvim#121]
Neovim 0.7.2, all plugins are at the latest versions.
When loading the nvim I get the following messages and the colors are completely off...

```
Error detected while processing /home/local/MAGICLEAP/ssavenko/.local/share/nvim/plugged/material.nvim/colors/material.lua:
W18: Invalid character in group name
W18: Invalid character in group name
W18: Invalid character in group name
```

**Comment by marko-cerovac** [comment]
Are there any other messages?
And can you please post a screenshot of the colors, so that I can diagnose the issue easier.

**Comment by trev-dev** [comment]
I'm on 0.7.0 and I have 24 of these warnings on startup but everything looks fine.  Wish I could tell you what broke.

**Comment by marko-cerovac** [comment]
I believe that this is an issue with the way treesitter handles highlight groups. They have recently changed highlight groups names to @GROUP_NAME  it is possible that this is illegal in nvim < 0.8

**Comment by trev-dev** [comment]
Fair enough.  On some distros "stable" neovim will be behind until the next release.  This may be the case for nix.

**Comment by marko-cerovac** [comment]
@trev-dev I've added a condition that checks for neovim version and applies treesitter highlights accordingly. Please let me know if the error still pops up.

**Comment by trev-dev** [comment]
Nice, thanks @marko-cerovac!!
